scripts/ci/selective_tests.sh

```diff
@@ -125,18 +182,60 @@ function count_changed_files() {
     echo "${count_changed_files}"
 }
 
-function run_all_tests_when_push_or_schedule() {
+function run_everything_when_push_or_schedule() {
     if [[ ${GITHUB_EVENT_NAME} == "push" || ${GITHUB_EVENT_NAME} == "schedule" ]]; then
         echo
         echo "Always run all tests in case of push/schedule events"
         echo
-        set_outputs_run_all_tests
-        exit
+        set_outputs_run_everything_and_exit
+    fi
+}
+
+function check_if_api_tests_should_be_run() {
+    local api_triggering_patterns=(
+        "^airflow/api"
+    )
+    pattern_array=("${api_triggering_patterns[@]}")
+    show_changed_files "$(get_regexp_from_patterns)"
```

Comment:
All of the `show_changes_files` are being passed an argument that isn't even used.
```suggestion
    show_changed_files
```

Reply:
Ah yeah. Cleaning it up